data/prefixes.js

```diff
@@ -779,3 +779,12 @@ f(over, { match: /a #1/ }, browsers =>
         browsers
     })
 );
+
+// CSS color-adjust property
+f(require('caniuse-lite/data/features/css-color-adjust.json'), browsers =>
+    prefix(['color-adjust'], {
+        mistakes: ['-khtml-', '-ms-', '-moz-', '-o-'],
```

Comment:
If I got it in the correct way, this array prevents Autoprefixer from adding unnecessary prefixes. In our case there is no need to add any prefixes except `-webkit-`.

Reply:
Not, it is not correct. Check other prefixes, they don’t have `mistakes`. `mistakes` is only for popular mistakes. I think nobody wrote `-khtml-color-adjust` in the real world.